apps/api/src/controllers/accounts.controller.ts

```diff
@@ -14,6 +15,20 @@ export class AccountsController {
     this.logger = new Logger(this.constructor.name);
   }
 
+  @Get('siwf')
+  @HttpCode(HttpStatus.OK)
+  @ApiOperation({ summary: 'Get the Sign-In With Frequency Configuration' })
+  @ApiOkResponse({ description: 'Returned SIWF Configuration data', type: WalletLoginConfigResponse })
+  async getSIWFConfig(): Promise<WalletLoginConfigResponse> {
+    try {
+      return await this.accountsService.getSIWFConfig();
```

Comment:
nit: `await` not necessary on return values when the function returns a Promise
```suggestion
      return this.accountsService.getSIWFConfig();
```